packages/components/src/Disclosure/Disclosure.css

```diff
@@ -0,0 +1,64 @@
+/*
+Re: (.arrowIconWrapper > svg > path)
+This ensure we are spinning just the svg graphic and not the
+conter, which result in an odd square rendering glitch when
+displaying the background color together.
+*/
+.arrowIconWrapper > svg > path {
+  transform-origin: 50% 50% 0;
+  transition: transform var(--timing-base);
+}
+
+.flippedVertical > svg > path {
+  transform: rotate(180deg);
+}
+
+.summary {
+  display: flex;
+  border: none;
+  list-style: none;
+  cursor: pointer;
+  outline-color: none;
```

Comment:
Can we add `transition: background var(--timing-quick), box-shadow(--var-timing-base)` so we get  a smoother transition on the hover/focus state changes?